src/components/ScenarioParameters/components/README.md

```diff
@@ -0,0 +1,99 @@
+# Add a new tab in Scenario Parameters
+
+- Open azure-sample-webapp/src/components/ScenarioParameters/component/BasicTypes.js
```

Comment:
We should probably remove the reference to the parent folder "azure-sample-webapp", because its name will be different in forked projects